api-description/forms/pk/post.json

```diff
@@ -0,0 +1,23 @@
+{
+    "description": "Endpoint to answer a form",
+    "url": "http://localhost:8000/forms/1",
+    "fields": {
+        "answers": {
+            "1": "Friends",
+            "2": "one_time",
+            "3": ["two_times", "three_times"],
+            "4": 5
```

Comment:
Just realised the return value of this endpoint might show this value as `"5"`.  I genuinely don't know yet, it depends on the internals of the process.  I'll change the description later if so.